package.json

```diff
@@ -117,18 +118,11 @@
     "test": "react-scripts test"
   },
   "homepage": ".",
-  "browserslist": {
-    "production": [
-      ">0.2%",
-      "not dead",
-      "not op_mini all"
-    ],
-    "development": [
-      "last 1 chrome version",
-      "last 1 firefox version",
-      "last 1 safari version"
-    ]
-  },
+  "browserslist": [
```

Comment:
what does browserslist do? does rest of app work fine after this change?

Reply:
> The browserslist configuration controls the outputted JavaScript so that the emitted code will be compatible with the browsers specified.  (https://create-react-app.dev/docs/supported-browsers-features/#configuring-supported-browsers)